general/clients/skins.md

```diff
@@ -0,0 +1,79 @@
+---
+uid: css-custom
+title: CSS Customization
+---
+
+# CSS Customization
+
+Custom CSS provides interface customization such as changing colors or modifying behavior. Here are a collection of community scripts available. Currently, the css modifications only work on WEBUI. To implement these changes, go to Dashboard > General > Custom CSS.
```

Comment:
```suggestion
Custom CSS provides interface customization such as changing colors or modifying behavior. Here are a collection of community scripts available. Currently, the CSS modifications only work on the web client. To implement these changes, go to Dashboard > General > Custom CSS.
```

Reply:
This will need to be reworded once https://github.com/jellyfin/jellyfin-web/pull/763 is flushed out.